python/lsst/analysis/tools/actions/plot/diaSkyPlot.py

```diff
@@ -82,15 +82,6 @@ class DiaSkyPlot(PlotAction):
         default={},
     )
 
-    def getInputSchema(self) -> KeyedDataSchema:
-        """Defines the schema this plot action expects (the keys it looks
-        for and what type they should be). In other words, verifies that
-        the input data has the columns we are expecting with the right dtypes.
-        """
-        for panel in self.panels.values():
-            yield (panel.ra, Vector)
-            yield (panel.dec, Vector)
-
     def __call__(self, data: KeyedData, **kwargs) -> Mapping[str, Figure] | Figure:
```

Comment:
We should probably still have a getInputSchema.

`for ra in panel.ras:
    yield (ra, Vector)
for dec in panel.decs:
    yield (dec, Vector)`

Reply:
Sorry, I failed to add this back, will fix